package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "tsc -p src",
     "prepublish": "npm run build",
-    "test": "mocha --reporter dot test/tokenizer.js test/parser.js && node test/run-cases.js",
```

Comment:
fwiw you can add a `test/mocha.opts` file that contains:
```
--reporter dot
--recursive
```
and then `mocha test` would work as you expect.

Reply:
`mocha test/` already works as I expect? I'm not sure what difference you're pointing to.

Reply:
you removed `--reporter dot`, my suggestion would restore it; using `--recursive` means you can have tests in subfolders and they'll Just Work.

Reply:
Ah: removing that was intentional, as was the decision to make it not recurse.